### DISCUSSION TITLE
Custom events in Electron/Tauri/NeutralinoJS/etc.

### DISCUSSION THREAD

**Discussion by stephanbogner** [plausible/analytics#2468] [Anything Else]
Hey there,

(First of all: Plausible is great! I love it :heart:)

I am working on a **cross-platform desktop app (using Tauri)** and I was wondering whether it's possible to **use custom events** to track stuff.

I found the [integration guide](https://plausible.io/docs/integration-guides) and discussions ([1](https://github.com/plausible/analytics/discussions/677) and [2](https://github.com/plausible/analytics/discussions/1161)) regarding mobile tracking, but surprisingly nothing about Electron, Tauri or NeutralinoJS (or I missed it).

**My assumption is that the recommended way is to use post-requests with [the API-endpoints](https://plausible.io/docs/events-api).**
Is that assumption correct?

**Comment by ukutaht** [comment]
Thanks for reaching out and the kind words @stephanbogner 

> My assumption is that the recommended way is to use post-requests with the API-endpoints.

Yes, that's correct.

**Reply by stephanbogner** [reply]
Wow, that was insanely fast and thanks a lot for confirming!

**Comment by jaads** [comment]
How to to get the user agent header value in tauri though? 🤔
